awx_collection/plugins/modules/token.py

```diff
@@ -85,9 +85,9 @@
         controller_username: "{{ my_username }}"
         controller_password: "{{ my_password }}"
 
-    - name: Use our new token to make another call
+    - name: Use our new token (via the Ansible facts system) to make another call
       job_list:
-        controller_oauthtoken: "{{ token }}"
+        controller_oauthtoken: "{{ ansible_facts.controller_token.token }}"
```

Comment:
So this is the real functional change. Does it not work with just `token`? Is this just to kind of clarify where it comes from?

Reply:
It looks like this functionality did actually change in commit 4add72b9d27116ddd0b5f478634a956e588ba8aa:
```
-        'tower_token': last_response,
+        'token': last_response,
```
However, we shouldn't need to reference it with ansible_facts. We should just be able to use `controller_token.token`.

Reply:
Also, you shouldn't have to reference the `.token`, the controller_api.py module already compensates for that. See https://github.com/ansible/awx/blob/devel/awx_collection/plugins/module_utils/controller_api.py#L118

Reply:
Thanks to both of you for your comments, sorry this fell through the notifications cracks during the holiday.
I will refresh my PR according to what you said and resubmit.

Reply:
Sorry for the lack of movement here, just to let you know that this is still on my todo list.

Reply:
@vrubiolo we are circling back to this, we are still interested in getting this doc change. Could you get a chance to make the change as described above? Once that in, we'll merge it

thanks